provision/ubuntu/containerd.sh

```diff
@@ -4,9 +4,9 @@ source "${ENV_FILEPATH}"
 
 set -e
 
-CONTAINERD_TARGZ=cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz
+CONTAINERD_TARGZ=cri-containerd-cni-${CONTAINERD_VERSION}-linux-amd64.tar.gz
 
-wget https://storage.googleapis.com/cri-containerd-release/${CONTAINERD_TARGZ}
+wget https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VERSION}/
```

Comment:
You're missing a file at the end. For example, it should be: `https://github.com/containerd/containerd/releases/download/v1.6.3/containerd-1.6.3-linux-amd64.tar.gz`. That's why the builds are currently failing.